src/devices/SocketCan/SafeCanRawSocketHandle.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.IO;
+using System.Net.Sockets;
+using System.Reflection;
+using System.Runtime.InteropServices;
+using System.Runtime.Serialization;
+
+namespace Iot.Device.SocketCan
+{
+    internal class SafeCanRawSocketHandle : SafeHandle
```

Comment:
I don't have much experience with SafeHandles, but shouldn't this have a finalizer that calls ReleaseHandle? Also, can we add xml docs for the class?

Reply:
He is subclassing Safehande which has the finalizer which will call ReleaseHandle

---
In reply to: [288320949](https://github.com/dotnet/iot/pull/438#discussion_r288320949) [](ancestors = 288320949)